copycat/file/src/main/java/org/apache/kafka/copycat/file/FileStreamSourceTask.java

```diff
@@ -68,18 +78,14 @@ public void start(Properties props) {
                     log.debug("Skipped to offset {}", lastRecordedOffset);
                 }
                 streamOffset = (lastRecordedOffset != null) ? lastRecordedOffset : 0L;
+                reader = new BufferedReader(new InputStreamReader(stream));
             } catch (FileNotFoundException e) {
-                throw new CopycatException("Couldn't find file for FileStreamSourceTask: {}", e);
+                log.warn("Couldn't find file for FileStreamSourceTask, sleeping to wait for it to be created");
+                Thread.sleep(1000);
```

Comment:
Should this sort of "backoff" be the responsibility of whoever is polling for data?